stroke/scoring.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, name="Sørensen–Dice Coefficient", precision=3):
         self.is_lower_the_better = False
         self.minimum = 0
         self.maximum = 1
+        self.worst = 0
```

Comment:
I doubt it's the right fix:

https://github.com/paris-saclay-cds/ramp-workflow/blob/master/rampwf/score_types/base.py#L10

is your scoring inheriting from BaseScoreType? it should (I think)